book/src/framework/components/troubleshooting.md

```diff
@@ -1,5 +1,15 @@
 # Troubleshooting
 
+## Can't start `ctf obs u`
+```
+Error response from daemon: error while creating mount source path '/host_mnt/Users/fsldkfs/Downloads/compose/conf/provisioning/dashboards/cadvisor/cadvisor.json': mkdir /host_mnt/Users/sdfjskj/Downloads/compose/conf/provisioning: operation not permitted
+exit status 1
+```
+
+#### Solution
+
+Use another directory with write access or change the directory access
```

Comment:
to be exact this solved the issue in question:
![image](https://github.com/user-attachments/assets/06d5846e-9b7f-4a6f-bf2e-a459731349d0)